src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -102,7 +102,7 @@ public override Task<HttpResponseMessage> ExecuteAsync(HttpControllerContext con
             // For all admin api requests, we'll update the ScriptHost debug timeout
             // For now, we'll enable debug mode on ANY admin requests. Since the Portal interacts through
             // the admin API this is sufficient for identifying when the Portal is connected.
-            _scriptHostManager.Instance.NotifyDebug();
+            _scriptHostManager.Instance?.NotifyDebug();
```

Comment:
Funny - this is the bug I'm fixing in https://github.com/Azure/azure-webjobs-sdk-script/pull/654